src/Installers/Windows/AspNetCoreModule-Setup/IIS-Common/build/compiler.props

```diff
@@ -0,0 +1,17 @@
+<Project>
+
+  <PropertyGroup>
+    <CppNugetVersion>14.11.25547</CppNugetVersion>
+    <CppNugetPath Condition="$(CppNugetPath) == '' AND Exists('$(UserProfile)\.nuget\packages\visualcpptools.community.vs2017layout\$(CppNugetVersion)\build\native\VisualCppTools.Community.VS2017Layout.props')">$(UserProfile)\.nuget\packages\visualcpptools.community.vs2017layout\$(CppNugetVersion)\build\native\VisualCppTools.Community.VS2017Layout.props</CppNugetPath>
```

Comment:
I think we talked about this one. Are we okay just using the in-box toolset instead of pulling a 640 MB package?

Reply:
Will try to; will need to verify on CI as last time we made that change we got mismatching compiler versions 😄